lib/create-icon-set-from-icomoon.js

```diff
@@ -0,0 +1,20 @@
+/**
+* @providesModule createIconSetFromIcoMoon
+* @flow
+*/
+'use strict';
+
+let createIconSet = require('./create-icon-set');
+
+function createIconSetFromIcoMoon(config : Object, fontFamily? : string, fontFile? : string) : Function {
+  var glyphMap = {};
+  config.icons.forEach(function (icon) {
+    glyphMap[icon.properties.name] = icon.properties.code;
+  });
+  fontFamily = fontFamily || config.name;
```

Comment:
Sorry, I don't understand. Do you mean that I should replace line14 with that expression?